firebase-installations/api.txt

```diff
@@ -44,6 +44,7 @@ package com.google.firebase.installations.local {
     method @NonNull public static com.google.firebase.installations.local.PersistedInstallationEntry.Builder builder();
     method @Nullable public abstract String getAuthToken();
     method public abstract long getExpiresInSecs();
+    method @Nullable public abstract String getFISException();
```

Comment:
I think this should be #getFisException

Compare:
http://go/java-style#s5.3-camel-case

Reply:
Done.thanks.